test/unit/sql/where.test.ts

```diff
@@ -2246,6 +2246,39 @@ describe(getTestDialectTeaser('SQL'), () => {
 
     if (sequelize.dialect.supports.JSONB) {
       describe('JSONB', () => {
+        if (dialectSupportsArray()) {
+          testSql({
+            jsonbAttr: {
+              [Op.anyKeyExists]: ['a', 'b'],
+            },
+          }, {
+            default: `[jsonbAttr] ?| ARRAY['a', 'b']`,
+          });
+
+          testSql({
+            jsonbAttr: {
+              [Op.allKeysExist]: ['a', 'b'],
+            },
+          }, {
+            default: `[jsonbAttr] ?& ARRAY['a', 'b']`,
+          });
+
+          testSql({
+            jsonbAttr: {
+              [Op.anyKeyExists]: literal(`SELECT jsonb_array_elements_text('ARRAY["a","b"]')`),
+            },
+          }, {
+            default: `[jsonbAttr] ?| ARRAY(SELECT jsonb_array_elements_text('ARRAY["a","b"]'))`,
+          });
+
+          testSql({
+            jsonbAttr: {
+              [Op.allKeysExist]: literal(`SELECT jsonb_array_elements_text('ARRAY["a","b"]')`),
```

Comment:
Same here

```suggestion
              [Op.allKeysExist]: literal(`ARRAY(SELECT jsonb_array_elements_text('ARRAY["a","b"]'))`),
```

Reply:
Since we will wrap the literal within an array, I think we will skip this commit suggestion https://github.com/sequelize/sequelize/pull/15073#discussion_r990766377

Reply:
Thanks